package.json

```
@@ -16,12 +16,13 @@
     "prepublishOnly": "npm test",
     "test": "ts-node test | tap-diff",
     "test:unit": "ts-node test/unit | tap-diff",
-    "test:e2e": "ts-node test/e2e"
+    "test:e2e": "ts-node test/e2e",
+    "postinstall": "npm run build"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@holochain/conductor-api": "^0.0.1-dev.8",
+    "@holochain/conductor-api": "^0.0.1-dev.10",
```

since conductor-api doesn't really use semver yet, for real version pinning it might be better to do this instead?
```suggestion
    "@holochain/conductor-api": "0.0.1-dev.10",
```

---

otherwise it will still just resolve to latest released version when someone `npm install`s regardless of breaking changes or not. Which could result in confusion, and the dreaded version mismatch headaches.

---

good idea, actually.